ModPatches/Vanilla Factions Expanded - Pirates/Defs/Vanilla Factions Expanded - Pirates/Ammo/Ammo_Mods.xml

```diff
@@ -36,6 +36,15 @@
 		</ammoTypes>
 	</CombatExtended.AmmoSetDef>
 
+	<!-- Bioferrites -->
+	<CombatExtended.AmmoSetDef>
+		<defName>AmmoSet_50mmRocket_DF</defName>
+		<label>50mm rocket(Direct fire)</label>
+		<ammoTypes>
+			<Ammo_50mmRocket_HE>Bullet_50mmRocket_HEDF</Ammo_50mmRocket_HE>
```

Comment:
We're probably going to want something smaller than the 50mm--if memory serves, these are stat'd after the S-5 rocket, which is 1.4 meters long, and it has the bulk to match. It'll probably have to be a custom ammo set.